package.json

```diff
@@ -65,6 +68,7 @@
     "tslib": "2.4.0",
     "typescript": "4.6.4",
     "vite": "3.1.0",
+    "vite-plugin-require": "^1.0.8",
```

Comment:
Same as above. I assume this dependency is to allow using `require` somewhere in the codebase? We should avoid this and stick to `import()`